esp/bindings/http/platform/httpservice.cpp

```diff
@@ -268,8 +313,16 @@ int CEspHttpServer::processRequest()
             ESPLOG(LogMin, "%s %s, from %s", method.str(), m_request->getPath(pathStr).str(), m_request->getPeer(peerStr).str());
         else //user ID is in HTTP header
             ESPLOG(LogMin, "%s %s, from %s@%s", method.str(), m_request->getPath(pathStr).str(), userid, m_request->getPeer(peerStr).str());
-
+//checkUserAuth could declare nested span
```

Comment:
Should probably also move to the unit tests.  Creating spans when we call ldap etc. is something worth recording in a jira to come back to later.